packages/main/src/TimePicker.ts

```diff
@@ -314,6 +313,12 @@ class TimePicker extends UI5Element implements IFormInputElement {
 	@query("[ui5-time-selection-clocks]")
 	_timeSelectionClocks?: TimeSelectionClocks;
 
+	@query("[ui5-popover]")
+	_inputsPopover?: Popover;
```

Comment:
Also consider declaring this property with `!` instead of `?` here so you don't have to type the `!` everywhere you use it later.